src/ocamlorg_frontend/components/toc.eml

```diff
@@ -15,12 +15,12 @@ let render (t : t) =
     >
     <% (match t with [] ->  %>
     <% | _ -> %>
-    <div class="font-semibold text-gray-500 text-sm mb-4">On This Page</div>
+    <div class="font-bold text-title dark:text-dark-title text-sm mb-4">On This Page</div>
     <ol class="leading-6 text-sm border-l">
       <% t |> List.iter begin fun item -> %>
         <li>
-          <a href="<%s item.href %>" class="text-legacy-default py-1 pl-2 pr-1 block transition-colors border-l-4 border-t-2 border-b-2 border-r-2 border-transparent rounded"
-            :class='isWithin(scrollY, "<%s item.href %>", sectionYPositions) ? "font-semibold border-l-primary text-primary bg-legacy-primary-200": "hover:text-primary"'
+          <a href="<%s item.href %>" class="text-title dark:text-dark-title py-1 pl-2 pr-1 block transition-colors border-l-4 border-t-2 border-b-2 border-r-2 border-transparent rounded"
```

Comment:
Also why are there different text classes for the links, it is `text-title` here and `text-content` below for the same `text-legacy-default` class